src/lib/components/CurrencyCalculator.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+	import { CURRENCIES } from '$lib/types/Currency';
+	import { toCurrency } from '$lib/utils/toCurrency';
+	import type { Currency } from '$lib/types/Currency';
+
+	$: firstDeviseValue = 1;
+	$: firstDevise = 'BTC' as Currency;
+	$: secondDevise = 'EUR' as Currency;
+	$: secondDeviseValue = toCurrency(secondDevise, firstDeviseValue, firstDevise);
```

Comment:
`Devise` is a french word, use `Currency` instead.

Use `amount` instead of `value`. Same as for `Price`